### ISSUE TITLE
Index code in search

### ISSUE THREAD

**Issue by loopDelicious** [postmanlabs/postman-docs#2445]
Currently if I search for code, like a Postman function, not all code displays in the results.

- `$randomBs` and other dynamic variables
- `pm.environment` displays the sandbox API reference, but not any of the pages where it's mentioned
-

**Comment by SueSmith** [comment]
@loopDelicious Aye so currently code excerpts are only indexed for search if we add the relevant terms manually to the frontmatter in each page.. would be good to automate this

**Comment by ch264** [comment]
I agree, automation would be great. We have discussed this ticket before and the entire conversation is in Jira ticket WEBSITES-137. 

Algolia React-Instantsearch does not support indexing code blocks out of the box. So by default they do not turn any content in a codeblock in < pre > and < code > tags into objects. Instead Algolia ignores it.


This is what Algolia sais: 
”Using <code/> tags to wrap long snippet of code will introduce some noise since these snippets are really similar from one page to another. It will introduce duplicates and wrongly interfere with the data indexed.” https://discourse.algolia.com/t/experience-and-advice-for-dealing-with-indexing-of-code-in-software-documentation/8307


so if we want to index mainly the objects and methods in the pm api.. Algolia's point is then that, for example there would be tons of results for “pm”. 

That is the reason we decided to include the most searched terms in the frontmatter of the page.
